control-plane/csi-driver/src/bin/node/main_.rs

```diff
@@ -309,6 +319,31 @@ pub(super) async fn main() -> anyhow::Result<()> {
         check_ana_and_label_node(&kube_client, node_name, nvme_enabled).await?;
     }
 
+    let conn_fallback = matches.get_flag("nvme-connect-fallback");
+    // Check and store if this node has an RDMA device so that nvme connect over RDMA
+    // can be done. If this node is also an io-engine node with rdma enabled and working,
+    // then this check will always set capability true.
+    let ibv_output = tokio::process::Command::new("ibv_devinfo")
```

Comment:
Can this be done with a library call instead?

Reply:
> Can this be done with a library call instead?

We could, there are some crates available not sure how stable. Until we start managing devices on our own and have the crates tested, I don't intend to integrate those in our code.

Reply:
There's a crate from a very well known peep in the rust community, but seems to build the library itself during the build, which might complicate things :(
Let's use the binary then.